src/main/java/appeng/client/me/ItemRepo.java

```diff
@@ -112,6 +113,8 @@ public void updateView()
 
 		final Enum viewMode = this.sortSrc.getSortDisplay();
 		final Enum searchMode = AEConfig.instance().getConfigManager().getSetting( Settings.SEARCH_MODE );
+		final boolean needsZeroCopy = viewMode == ViewItems.CRAFTABLE;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "needsZeroCopy" closer to the code that uses it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1941)